content/docs/atom-guide/look-dev/tools/material-editor.md

```diff
@@ -0,0 +1,90 @@
+- From the Asset Browser, double click on a ".material" or ".materialtype".
+- From the Asset Browser, right click on a ".material" or ".materialtype", then select **Open in Material Editor...** 
+- Material Editor is also a standalone executable that can be launched directly from a file browser or command console.
+  - Launch the executable `<build>\bin\profile\MaterialEditor.exe`
+  - This requires passing the �project-path, followed by the path your project, as a command line argument.
+
+### Creating Materials
+You have multiple options to start editing a material.
+- Create a new material from a material type.
+  - Select **Main Menu > File > New...**.
+    - This opens the "Create Material Document" dialog.
+  - Select a material type as the basis for your new material.
+  - Select a path and file name for the new material.
+- Open an existing material
+  - Select **Main Menu > File > Open...**.
+- Open or create a material from the Asset Browser.
+
+### Editing Materials
+- All material editing is done by changing properties in the Material Editor inspector.
+- Changes are recorded as properties are edited and can be undone and redone. 
+
+### Child Materials
+- Materials can reference a parent material to inherit and override its properties.
+- Changes made to the parent, or another ancestor, are automatically reflected in the child material unless the child is overriding those properties.
+- Child materials can be created opening an existing material and selecting **Main Menu > File > Save As Child...**. 
+
+### Mirroring Changes in The O3DE Editor
+- Materials saved in the Material Editor will be processed by the Asset Processor.
+- If the O3DE Editor is open and the saved material is assigned to a Material Component or terrain it will automatically not reload to reflect the changes.
+- Enabling the autosave in Material Editor settings dialog will causes updates to be continuously mirrored in the O3DE Editor.
+
+## Navigating Material Editor
+After Material Editor has opened and initialized it will display the main window, as shown below.
+
+![Material Editor](/images/atom-guide/tools/material-editor.jpg)
+
+For more information about features common to Atom tools, menus, dockable panels, and working with documents, see [Atom Tools Common Features](/docs/atom-guide/look-dev/tools/atom-tools-common-features/). 
+
+### Document Types and Views
+Material Editor supports creating, opening, and editing one type of document.
+
+#### Material Documents
+Material Documents are the main documents type in Material Editor.
+- ".material" files can be opened and edited.
+- ".materialtype" files can be opened but cannot be saved.
+- The material being edited by the active material document will be displayed on the model in the viewport. 
+- As changes are made to the material through the inspector or script, the material shown in the viewport will automatically updated to reflect the changes.
```

Comment:
@gadams3 I think some ideas here can be tied together more cohesively. 
- Can you verify if the sentence about `materialtype` is accurate? The way I understand this is you can only read the file (through Material Editor?), probably for the purpose of knowing what's in it. We're allowed to open it in a text file and edit it there, right? That doesn't need to be included here, just wanted to know. 
- Based on my understanding of these sections, I propose reformatting the sections like this: 
```
## Navigating Material Editor

### Document types

### Views

#### Main menu

#### Viewport

#### Viewport Settings
```


```suggestion
### Document types
Material Editor supports material documents, `.material` and `.materialtype`. You can create, open, and edit `.material files`.  You can also open `.materialtype` files to view it's properties, but you cannot edit them in the Material Editor. 

The material that you're currently editing displays on the model in the viewport. As you edit the material through the inspector or script, the material shown in the viewport automatically updates to reflect the changes.
```

Reply:
This is mostly correct. Technically you can open and edit a material type as a material document. You can edit the properties and visualize the changes, just as you would a material file. You cannot save a material type from the material editor. However, you can open one, make some changes, and use save as child to save it as a material.

Reply:
Oh gotcha, I see. So how about this, or something similar:
> You can also open `.materialtype` files, edit its properties, and save it as a new material by choosing **Save As Child...**.